addon-test-support/-private/is-focusable.js

```diff
@@ -0,0 +1,10 @@
+export default function isFocusable(el) {
```

Comment:
Same logic is in focus and blur, + if we want to mimic specification, it might be much more logic actually